README.rst

```diff
@@ -283,8 +284,13 @@ by applying labels to them, like this:
 
 GitHub creates the ``bug``, ``enhancement``, and ``documentation`` labels for you.
 Dependabot creates the ``dependencies`` label.
-Create the remaining labels when you need them,
-on the *Issues* tab of your GitHub repository,
+
+Project labels are managed automatically by `GitHub Labeler`_
+each time you push to your main branch.
+The labels are defined in ``.github/labels.yml`` file.
+
+You can still manage labels manually on the *Issues* tab of your GitHub repository.
+GithHub Labeler is set by default to keep the labels not defined in the YAML file.
```

Comment:
Hm... This is a lot of detail, especially since the text appears verbatim in the README.rst, the Quickstart Guide, and the User Guide.

Actually, I would prefer to remove the entire text after the table, starting from here:

> GitHub creates the ``bug``, ``enhancement``, and ``documentation`` labels for you.
> Dependabot creates the ``dependencies`` label.

Now that GitHub Labeler ensures that all labels are present, this information is irrelevant. 

I would also suggest to remove the text that you added:

```suggestion
```

This level of detail is not needed in the README and the Quickstart Guide. In the User Guide it duplicates information in the section "The Labeler Workflow", which comes right after this.

What do you think?

Reply:
Yes it make sense.